src/app/shared/services/apis/gb3/gb3-print.service.ts

```diff
@@ -177,6 +180,35 @@ export class Gb3PrintService extends Gb3ApiService {
   }
 
   private createGb2WmsPrintItem(activeMapItem: ActiveMapItem, gb2WmsSettings: Gb2WmsSettings): PrintMapItem {
+    const customLayerParameters: {[index: string]: string} = {};
+    if (gb2WmsSettings.filterConfigurations) {
+      const attributeFilterParameters = this.gb3TopicsService.transformFilterConfigurationToParameters(
+        gb2WmsSettings.filterConfigurations ?? [],
+      );
+      attributeFilterParameters.forEach((filterParameter) => {
+        customLayerParameters[filterParameter.name] = filterParameter.value;
+      });
+    }
+    if (gb2WmsSettings.timeSliderConfiguration) {
```

Comment:
You can get rid of the `!` after `gb2WmsSettings.timeSliderExtent!` by using the following suggestion
```suggestion
    if (gb2WmsSettings.timeSliderConfiguration && gb2WmsSettings.timeSliderExtent) {
```